test/Concurrency/Runtime/async_taskgroup_discarding_dontLeak_class_error.swift

```diff
@@ -12,35 +12,75 @@
 
 import _Concurrency
 
+actor SimpleCountDownLatch {
+  let from: Int
+  var count: Int
+
+  var continuation: CheckedContinuation<Void, Never>?
+
+  init(from: Int) {
+    self.from = from
+    self.count = from
+  }
+
+  func hit() {
+    defer { count -= 1 }
+    if count == 0 {
+      fatalError("Counted down more times than expected! (From: \(from))")
+    } else if count > 0 {
```

Comment:
This should probably be `count == 1` rather than `count > 0`.

Reply:
Thank you, fixed